### PR TITLE
Add missing requires to auoms package

### PR DESCRIPTION
**Pull request by dmcilvaney**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

The `auoms` package creates the file `/etc/init.d/auoms`, however the `chkconfig` package explictly claims ownership of the `/etc/init.d` directory. If `chkconfig` is installed after `auoms` it will find that directory already present and will fail to install (this ordering is common since package lists are alphabetical). Claim `chkconfig` as a runtime requirement for `auoms` so that the `chkconfig` package can create the directory as expected. Also add `procps-ng` and `initscripts` packages since post and preun stpeps call into 
`auomsctl` which relies on `pgrep` and `service`.

###### Change Log  

- Add `chkconfig` as a runtime requirement for `auoms`
- Add `procps-ng` as runtime requirement for `auoms`
- Add `initscripts` as runtime requirement for `auoms`

###### Does this affect the toolchain?  

NO

###### Test Methodology

- Local builds
- Manual install/uninstall on VMs

### CODE REVIEW

**Comment by dmcilvaney** [comment]
Linted spec doesn't install correctly, looking into it.

**Comment by dmcilvaney** [comment]
Actual issue was missing initscripts requirement

**Review by henryli001 — APPROVED** [review]

File: SPECS/auoms/auoms.spec [hunk 1]
```diff
@@ -53,8 +53,8 @@ cp %{SOURCE3} ./
 %build
 grep AUOMS_BUILDVERSION auoms.version | head -n 4 | cut -d'=' -f2 | tr '\n' '.' | sed 's/.$//' | sed 's/^/#define AUOMS_VERSION "/' > auoms_version.h
 sed -i 's/$/"/' auoms_version.h
-cp -R /usr/include/boost /usr/local/include/boost
-mv /usr/include/boost /usr/include/boost148
+cp -R %{_includedir}/boost %{_prefix}/local/include/boost
+mv %{_includedir}/boost %{_includedir}/boost148
```
Comment (PawelWMS) [inline SPECS/auoms/auoms.spec:57]: Please don't modify the path for these two lines. The macros use here are related to the location on the final system where the package gets installed, and they are independend of where these files are located for the package build, so they are not the same thing. In theory we might change our macros at some point if we needed a different folder structure (unlikely) or the package might get modified and these changes should be independent of each other.

**Review by PawelWMS — CHANGES_REQUESTED** [review]

File: SPECS/auoms/auoms.spec (hunk 1 above)
Comment (PawelWMS) [reply SPECS/auoms/auoms.spec:57]: Discussed offline, resolving.

Comment (dmcilvaney) [reply SPECS/auoms/auoms.spec]: Reverted the destination paths, but left the source paths unchanged. The boost package which provides these files uses the %{_includedir} macro so in theory they should change in lockstep.

**Review by PawelWMS — APPROVED** [review]

**Comment by dmcilvaney** [comment]
Ignoring remaining SPEC linter suggestions. Whitespace is a bug, and the build expects the boot libraries to be placed in those exact folders, should not use a macro for the destination.

**Pull request merged**